go-packages/meep-net-char-mgr/algo-segment.go

```diff
@@ -89,6 +89,7 @@ type SegAlgoFlow struct {
 	CurrentThroughput             float64 //measured
 	CurrentThroughputEgress       float64 //measured
 	Path                          *SegAlgoPath
+	NewPath                       bool
```

Comment:
Recommend renaming flag to UpdateRequired or  IsNewPath

Reply:
renamed to UpdateRequired